spec/requests/vendor_api/v1.1/get_multiple_applications_spec.rb

```diff
@@ -0,0 +1,264 @@
+require 'rails_helper'
+
+RSpec.describe 'Vendor API - GET /api/v1.1/applications', type: :request do
+  include VendorAPISpecHelpers
+  include CourseOptionHelpers
+
+  before do
+    stub_const('VendorAPI::VERSION', '1.1')
+  end
+
+  it 'returns applications of the authenticated provider' do
```

Comment:
do we need to duplicate all those tests, I think maybe we should only be testing pagination, what do you think?